trackhubs/utils.py

```diff
@@ -0,0 +1,18 @@
+import ast
+import json
+
+
+def str2obj(var):
+    """
+    Convert string dict/array to an actual dict/array
+    this function is used to convert trackdb_configuration
+    and trackdb_data from string (as stored in MySQL) to JSON
+    (as it need for it to be stored in Elasticsearch when enriching the document)
+    :returns: JSON object
+    """
+    if isinstance(var, dict) or isinstance(var, list):
+        return var
```

Comment:
I don't remember coming across such a situation, that condition is there in case `var` is already a dictionary/list there is no need to proceed, simply return `var`.